lib/msf/core/post/file.rb

```diff
@@ -506,14 +506,15 @@ def rm_rf(*remote_dirs)
   # @param old_file [String] Remote file name to move
   # @param new_file [String] The new name for the remote file
```

Comment:
Could you please update the documentation to include that the return value is a boolean?

Reply:
sure :+1: